tests/integrations/current/test_write.py

```diff
@@ -13,22 +13,23 @@
 # limitations under the License.
 
 
+import collections
 import os
-import time
 import pytest
-import string
+import re
 import shutil
+import string
+import time
 
-from tests.integrations.base import BaseTest, pull
+from tests.integrations.base import BaseTest, pull, gitfs_log
```

Comment:
It's pytest fixture magic. For any test that takes an argument named like the fixture pytest will take care of instantiating the fixture and passing it in.
https://pytest.org/latest/fixture.html#fixtures-as-function-arguments